verify/cert.go

```diff
@@ -155,7 +176,10 @@ func (cv *CertVerifier) getConfDeepBlockNumber(ctx context.Context) (*big.Int, e
 }
 
 // retrieveBatchMetadataHash retrieves the batch metadata hash stored on-chain at a specific blockNumber for a given batchID
-// returns an error if some problem calling the contract happens, or the hash is not found
+// returns an error if some problem calling the contract happens, or the hash is not found.
+// We make an eth_call to the EigenDAServiceManager at the given blockNumber to retrieve the hash.
+// Therefore, make sure that blockNumber is <128 blocks behind the latest block, to avoid requiring an archive node.
```

Comment:
I'm not understanding where 128 is coming from. Is there some other value == 64 that gets added to `EthConfirmationDepth` to produce 128? Or is the comment just stale?

Reply:
See https://github.com/Layr-Labs/eigenda-proxy/pull/241#discussion_r1927216570